app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "409851798978",
```

Comment:
How many of these values need to be real? Can we check in fake values for now, then fill in the correct values once we have a shared developer-only Firebase project to reference? I plan to create both a developer and production project to make this simpler.

Reply:
From what I have seen, the package name should be the same and others can be changed. The only downside I see is that it will throw a lot of errors in the logcat when we use the dummy values, will that be okay ?

Reply:
Are these errors just at startup, or repeated? It's not ideal, and we should move to a debug project soon but some short-term errors just on app startup would be fine.

Can you file issues for adding the developer & production projects, and assign them to me? I defer to @vinitamurthi when these should be done by, but I'll make sure my part is done within whatever milestone we bucket them into.

Reply:
They are repeated errors, not just at the startup. Okay, I'll create those issues.

Reply:
@BenHenning I think we should allow this PR to go in with the fake data because all of the other PRs that @Sarthak2601 is going to create for this milestone is dependent on this PR (the build fails without this json so there has to be a json in for any of the firebase work to begin). So either we get the development project with the proper data ready ASAP, or we let this PR go in and then you can get the development project ready by maybe 1 week before milestone 1 ends.

Reply:
@vinitamurthi will it be fine to get this checked in early next week? I can get the development project created next Monday to unblock this PR. The alternative seems to be pretty bad: repeated errors in logcat.

Reply:
Resolving this since we the new project has been created